crowbar_framework/spec/lib/crowbar/checks/network_spec.rb

```diff
@@ -71,7 +71,15 @@
     it "returns true if fqdn is pingable" do
       allow_any_instance_of(Kernel).to(
         receive(:system).with(
-          "ping -c 1 #{subject.fqdn} > /dev/null 2>&1 || "\
+          "ping -c 1 #{subject.fqdn} > /dev/null 2>&1"
+        ).and_return(true)
+      )
+      expect(subject.ping_succeeds?).to be true
+    end
+    it "returns true if fqdn is pingable and ipv6" do
+      allow(subject).to receive(:ipv6_addrs).and_return(["fd00:0:0:3::1"])
```

Comment:
What is the meaning of this IPv6 addr here? I hope we dont actually use these, because the 40 bits following "FD" are supposed to be random.

Reply:
@bmwiedemann This is in the _spec.rb file which are unittests afaik